.ansible-lint

```diff
@@ -0,0 +1,16 @@
+exclude_paths:
+    - releasenotes/
+    - ../
+parseable: true
+quiet: false
+rulesdir:
+    - .ansible-lint_rules/
+verbosity: 1
+# Mocking modules is not recommended as it prevents testing of invalid
+# arguments or lack of their presence at runtime. It is preffered to
+# make use of requirements.yml to declare them.
+# mock_roles:
+mock_modules:
+    - baremetal_nodes_validate
```

Comment:
you can also remove this because I don't see any usage of the baremetal_nodes_validate module in the playbooks in this repo atm.

Reply:
+1

Reply:
Item removed.